### PR TITLE
Fix: removing ambiguities for compiling on Windows

### PR DESCRIPTION
**Pull request by ComicAddict**
Changes:

- Removing the ambiguous call for path declaration, resolved compiler errors
- Changing numeric limits to use the implicit macros in the standard library

### CODE REVIEW

File: src/io.cpp [hunk 1]
```diff
@@ -379,7 +381,7 @@ bool save_result_msh(const std::string& filename,
                      const std::vector<std::vector<size_t>>& shells,
                      const std::vector<std::vector<size_t>>& cells)
 {
-    constexpr size_t INVALID = std::numeric_limits<size_t>::max();
+    constexpr size_t INVALID = SIZE_MAX;
```
Comment (qnzhou) [inline src/io.cpp:384]: Hmm, what error does `std::numeric_limits<size_t>::max()` cause?

Comment (ComicAddict) [reply src/io.cpp:384]: I was getting compiler errors, ideally the macro definition for the `size_t`, i.e. `SIZE_MAX` is the same as the function call `std::numeric_limits<size_t>::max()`.

See the related post on [stack overflow](https://stackoverflow.com/questions/40492414/why-does-stdnumeric-limitslong-longmax-fail).

Comment (duxingyi-charles) [reply src/io.cpp:384]: looks good to me.

Comment (ComicAddict) [reply src/io.cpp]: Updated!

**Review by qnzhou — APPROVED** [review]
Thanks Tolga!

**Pull request merged**